src/Exception/ProblemDetailsExceptionInterface.php

```diff
@@ -18,21 +18,23 @@
  */
 interface ProblemDetailsExceptionInterface extends JsonSerializable, Throwable
 {
-    public function getStatus() : int;
+    public function getStatus(): int;
 
-    public function getType() : string;
+    public function getType(): string;
 
-    public function getTitle() : string;
+    public function getTitle(): string;
 
-    public function getDetail() : string;
+    public function getDetail(): string;
 
-    public function getAdditionalData() : array;
+    public function getAdditionalData(): array;
 
     /**
      * Serialize the exception to an array of problem details.
      *
      * Likely useful for the JsonSerializable implementation, but also
      * for cases where the XML variant is desired.
      */
-    public function toArray() : array;
+    public function toArray(): array;
+
+    public function jsonSerialize(): array;
```

Comment:
This is a fix for PhpUnit Mock. I'm not sure if I did something wrong, but without this it complaints about not being able to create a mock for `jsonSerialize` because it's an abstract method.

@weierophinney any idea on how to fix this properly?